tests/cupy_tests/test_cusparse.py

```diff
@@ -856,5 +913,12 @@ def test_sparseToDense(self, dtype):
             x = sparse.csc_matrix(x)
         elif self.format == 'coo':
             x = sparse.coo_matrix(x)
-        y = cusparse.sparseToDense(x)
+        try:
+            y = cusparse.sparseToDense(x)
+        except ValueError as e:  # 0-size matrices
+            if runtime.is_hip:
+                assert 'hipSPARSE' in str(e)
+                pytest.xfail('may be buggy')
+            else:
+                raise
```

Comment:
```suggestion
            raise
```
Should we always raise here? Both in the HIP and non-HIP case.

Reply:
Good idea, thanks Simon!